### PR TITLE
New layout goodies

### PR DESCRIPTION
**Pull request by notdryft**
Screenshot 2024-02-12 at 17 35 37

### CODE REVIEW

File: assets/js/index.ts [hunk 1]
```diff
@@ -131,54 +93,10 @@ const searchOptionsQueryFields = (query: string): SearchQuery[] =>
...
-      limit: searchLimit,
-      where: {
-        latest: "true",
-      },
-    });
-  }
+  return index.search({
+    query,
+    limit: searchLimit,
+  });
```
Comment (Isammoc) [inline assets/js/index.ts:99]:
```suggestion
const search = (query: string): Promise<FrontMatter[]> => index.search({
  query,
  limit: searchLimit,
});
```

File: README.md
Comment (Isammoc) [inline README.md:1]: Does that mean that we lose current SEO?
That existing links will be redirected to the main page?

**Review by Isammoc — APPROVED** [review]
One question + one nit

**Review by tpetillot — APPROVED** [review]

File: README.md
Comment (notdryft) [reply README.md:1]: Not entirely, the sitemap will be updated with the new links. Old pages will 404 though, the redirects (on nginx) are not planned to be updated (but can be as we have some days until we move to docs.gatling.io)

Comment (Isammoc) [reply README.md:1]: I didn't write about internal links. But external existing links that point to our site (stack overflow, jira answers, blog articles, etc.)
So, I do think that redirects (on nginx) should be updated to our new url scheme

Comment (notdryft) [reply README.md:1]: It's doable but they will become obsolete once we move to hubspot and there is no plan to have such redirects in hubspot to begin with.

**Pull request merged**